src/lsp/cobol_parser/grammar_expect.mli

```diff
@@ -0,0 +1,32 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                        SuperBOL OSS Studio                             *)
+(*                                                                        *)
+(*  Copyright (c) 2022-2023 OCamlPro SAS                                  *)
+(*                                                                        *)
+(* All rights reserved.                                                   *)
+(* This source code is licensed under the GNU Affero General Public       *)
+(* License version 3 found in the LICENSE.md file in the root directory   *)
+(* of this source tree.                                                   *)
+(*                                                                        *)
+(**************************************************************************)
+
+open Grammar.MenhirInterpreter
+
+type completion_entry =
+  | K of token Cobol_common.Basics.NEL.t
+  | QualifiedRef
+  | ProcedureRef
+
+val pp_completion_entry: completion_entry Fmt.t
+
+module CompEntrySet : (Set.S with type elt = completion_entry)
```

Comment:
Do you really need to expose that module? I suggest to simply expose `compare_completion_entry` (and maybe export a module `Completion_entry: Set.OrderedType with type t = completion_entry`), and then use `Set.Make` whenever necessary. That way other structures (maps, etc) can be defined.